labs/RLSS19_TextWorld.ipynb

```diff
@@ -0,0 +1,2871 @@
+{
```

Comment:
<p>I would put</p><p><code>!pip install git+https://github.com/Shmuma/TextWorld &gt; /dev/null 2&gt;&amp;1</code></p><p><code>!pip install ptan &gt; /dev/null 2&gt;&amp;1</code></p><p><code>!pip3 install torch torchvision &gt; /dev/null 2&gt;&amp;1</code></p><p>in one cell (<code> &gt; /dev/null 2&gt;&amp;1</code>makes the logs silent).</p><p>And then</p><p><code># Juggling around colab requirement</code></p><p><code>!pip uninstall -y prompt-toolkit &gt; /dev/null 2&gt;&amp;1</code></p><p><code>!pip install prompt-toolkit==1.0.16</code></p><p><code># DO NOT FORGET to restart runtime as suggested</code></p><p>in a next cell, with non-silent logs for the last install in order to have the prompt "RESTART RUNTIME" show up</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yfletberliac/rlss2019-hands-on/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>